PBRVulkan/RayTracer/src/Tracer/Menu.h

```diff
@@ -31,7 +31,7 @@ namespace Tracer
 		int MaxDepth = 3;
 		int SSP = 1;
 		float Fov{};
-		float Aperture = 0.1f;
+		float Aperture = 0.001f;
```

Comment:
Is there a particular reason for this change?

Reply:
Now it is used in the lens simulation.

Reply:
The question is more about the quantity? Why do you pick 0.001f?